2021-06-11 04:26:58 info: ---------------------------------------
2021-06-11 04:26:58 info: >>> Running Unit Test `ex1-johnWho` <<<
2021-06-11 04:26:58 info: ---------------------------------------
2021-06-11 04:26:58 info: Exercise has not yet been modified
2021-06-11 04:27:02 error: *** Unit Test Error Report ***

- getAnonName should call `new Promise()`
- getAnonName should take a single argument
- getAnonName : `resolve()` should be called with a one argument
- getAnonName : `reject()` should be called with a one argument
- getAnonName should resolve when called with a string argument,
- getAnonName should reject with an Error object when called without an argument,

2021-06-11 04:43:12 info: ---------------------------------------
2021-06-11 04:43:12 info: >>> Running Unit Test `ex1-johnWho` <<<
2021-06-11 04:43:12 info: ---------------------------------------
2021-06-11 04:43:18 error: *** Unit Test Error Report ***

- getAnonName should reject with an Error object when called without an argument
expect(received).rejects.toBeInstanceOf()

Received promise resolved instead of rejected
Resolved to value: "undefined Doe",

2021-06-11 04:44:12 info: ---------------------------------------
2021-06-11 04:44:12 info: >>> Running Unit Test `ex1-johnWho` <<<
2021-06-11 04:44:12 info: ---------------------------------------
2021-06-11 04:44:16 info: All unit tests passed.
2021-06-11 04:44:17 info: All steps were completed successfully
2021-06-11 04:50:44 info: ---------------------------------------
2021-06-11 04:50:44 info: >>> Running Unit Test `ex1-johnWho` <<<
2021-06-11 04:50:44 info: ---------------------------------------
2021-06-11 04:50:49 info: All unit tests passed.
2021-06-11 04:50:51 info: All steps were completed successfully
2021-06-14 12:47:36 info: ---------------------------------------
2021-06-14 12:47:36 info: >>> Running Unit Test `ex1-johnWho` <<<
2021-06-14 12:47:36 info: ---------------------------------------
2021-06-14 12:47:41 info: All unit tests passed.
2021-06-14 12:47:43 info: All steps were completed successfully
2021-06-14 12:56:58 info: -------------------------------------------------
2021-06-14 12:56:58 info: >>> Running Unit Test `ex2-checkDoubleDigits` <<<
2021-06-14 12:56:58 info: -------------------------------------------------
2021-06-14 12:57:02 error: *** Unit Test Error Report ***

- checkDoubleDigits (5) should return a rejected promise with an Error object
expect(received).rejects.toBeInstanceOf()

Received promise resolved instead of rejected
Resolved to value: "This is a double digit number!",
- checkDoubleDigits (123) should return a rejected promise with an Error object
expect(received).rejects.toBeInstanceOf()

Received promise resolved instead of rejected
Resolved to value: "This is a double digit number!",

2021-06-14 12:59:17 info: -------------------------------------------------
2021-06-14 12:59:17 info: >>> Running Unit Test `ex2-checkDoubleDigits` <<<
2021-06-14 12:59:17 info: -------------------------------------------------
2021-06-14 12:59:20 error: *** Unit Test Error Report ***

- checkDoubleDigits (10) should return a promise that resolves to "This is a double digit number!"
expect(received).resolves.toEqual()

Received promise rejected instead of resolved
Rejected to value: [Error: Expected a double digit number but got `number`],
- checkDoubleDigits (99) should return a promise that resolves to "This is a double digit number!"
expect(received).resolves.toEqual()

Received promise rejected instead of resolved
Rejected to value: [Error: Expected a double digit number but got `number`],

2021-06-14 01:02:36 info: -------------------------------------------------
2021-06-14 01:02:36 info: >>> Running Unit Test `ex2-checkDoubleDigits` <<<
2021-06-14 01:02:36 info: -------------------------------------------------
2021-06-14 01:02:39 error: *** Unit Test Error Report ***

- checkDoubleDigits (5) should return a rejected promise with an Error object
expect(received).rejects.toBeInstanceOf()

Received promise resolved instead of rejected
Resolved to value: "This is a double digit number!",

2021-06-14 01:03:17 info: -------------------------------------------------
2021-06-14 01:03:17 info: >>> Running Unit Test `ex2-checkDoubleDigits` <<<
2021-06-14 01:03:17 info: -------------------------------------------------
2021-06-14 01:03:21 error: *** Unit Test Error Report ***

- checkDoubleDigits (10) should return a promise that resolves to "This is a double digit number!"
expect(received).resolves.toEqual()

Received promise rejected instead of resolved
Rejected to value: [Error: Expected a double digit number but got `number`],
- checkDoubleDigits (123) should return a rejected promise with an Error object
expect(received).rejects.toBeInstanceOf()

Received promise resolved instead of rejected
Resolved to value: "This is a double digit number!",

2021-06-14 01:04:52 info: -------------------------------------------------
2021-06-14 01:04:52 info: >>> Running Unit Test `ex2-checkDoubleDigits` <<<
2021-06-14 01:04:52 info: -------------------------------------------------
2021-06-14 01:04:55 error: *** Unit Test Error Report ***

- checkDoubleDigits (10) should return a promise that resolves to "This is a double digit number!"
expect(received).resolves.toEqual()

Received promise rejected instead of resolved
Rejected to value: [Error: Expected a double digit number but got `number`],
- checkDoubleDigits (99) should return a promise that resolves to "This is a double digit number!"
expect(received).resolves.toEqual()

Received promise rejected instead of resolved
Rejected to value: [Error: Expected a double digit number but got `number`],
- checkDoubleDigits (123) should return a rejected promise with an Error object
expect(received).rejects.toBeInstanceOf()

Received promise resolved instead of rejected
Resolved to value: "This is a double digit number!",

2021-06-14 01:07:10 info: -------------------------------------------------
2021-06-14 01:07:10 info: >>> Running Unit Test `ex2-checkDoubleDigits` <<<
2021-06-14 01:07:10 info: -------------------------------------------------
2021-06-14 01:07:13 info: All unit tests passed.
2021-06-14 01:07:15 info: All steps were completed successfully
2021-06-14 01:26:25 info: ----------------------------------------
2021-06-14 01:26:25 info: >>> Running Unit Test `ex3-rollDice` <<<
2021-06-14 01:26:25 info: ----------------------------------------
2021-06-14 01:26:28 info: All unit tests passed.
2021-06-14 01:26:30 info: All steps were completed successfully
2021-06-14 02:38:01 info: --------------------------------------------
2021-06-14 02:38:01 info: >>> Running Unit Test `ex4-pokerDiceAll` <<<
2021-06-14 02:38:01 info: --------------------------------------------
2021-06-14 02:38:04 info: All unit tests passed.
2021-06-14 02:38:06 info: All steps were completed successfully
2021-06-14 02:38:29 info: --------------------------------------------
2021-06-14 02:38:29 info: >>> Running Unit Test `ex4-pokerDiceAll` <<<
2021-06-14 02:38:29 info: --------------------------------------------
2021-06-14 02:38:32 info: All unit tests passed.
2021-06-14 02:38:34 info: All steps were completed successfully
2021-06-14 02:38:45 info: --------------------------------------------
2021-06-14 02:38:45 info: >>> Running Unit Test `ex4-pokerDiceAll` <<<
2021-06-14 02:38:45 info: --------------------------------------------
2021-06-14 02:38:47 error: *** Unit Test Error Report ***

- ex4-pokerDiceAll should resolve when all dices settle successfully
expect(received).resolves.toBeDefined()

Received promise rejected instead of resolved
Rejected to value: [TypeError: undefined is not iterable (cannot read property Symbol(Symbol.iterator))],

2021-06-14 03:14:23 info: --------------------------------------------
2021-06-14 03:14:23 info: >>> Running Unit Test `ex4-pokerDiceAll` <<<
2021-06-14 03:14:23 info: --------------------------------------------
2021-06-14 03:14:27 info: All unit tests passed.
2021-06-14 03:14:29 info: All steps were completed successfully
2021-06-14 03:14:40 info: --------------------------------------------
2021-06-14 03:14:40 info: >>> Running Unit Test `ex4-pokerDiceAll` <<<
2021-06-14 03:14:40 info: --------------------------------------------
2021-06-14 03:14:41 info: All unit tests passed.
2021-06-14 03:14:43 info: All steps were completed successfully
2021-06-14 03:18:47 info: --------------------------------------------
2021-06-14 03:18:47 info: >>> Running Unit Test `ex4-pokerDiceAll` <<<
2021-06-14 03:18:47 info: --------------------------------------------
2021-06-14 03:18:51 info: All unit tests passed.
2021-06-14 03:18:51 error: *** ESLint Report ***

ex4-pokerDiceAll.js
  28:9  error  'dices' is assigned a value but never used  no-unused-vars

âœ– 1 problem (1 error, 0 warnings)


2021-06-14 03:19:11 info: --------------------------------------------
2021-06-14 03:19:11 info: >>> Running Unit Test `ex4-pokerDiceAll` <<<
2021-06-14 03:19:11 info: --------------------------------------------
2021-06-14 03:19:14 info: All unit tests passed.
2021-06-14 03:19:16 info: All steps were completed successfully
2021-06-14 03:37:34 info: --------------------------------------------
2021-06-14 03:37:34 info: >>> Running Unit Test `ex4-pokerDiceAll` <<<
2021-06-14 03:37:34 info: --------------------------------------------
2021-06-14 03:37:37 info: All unit tests passed.
2021-06-14 03:37:39 info: All steps were completed successfully
2021-06-14 03:44:39 info: ----------------------------------------
2021-06-14 03:44:39 info: >>> Running Unit Test `ex3-rollDice` <<<
2021-06-14 03:44:39 info: ----------------------------------------
2021-06-14 03:44:42 info: All unit tests passed.
2021-06-14 03:44:44 info: All steps were completed successfully
2021-06-15 11:50:25 info: ----------------------------------------
2021-06-15 11:50:25 info: >>> Running Unit Test `ex3-rollDice` <<<
2021-06-15 11:50:25 info: ----------------------------------------
2021-06-15 11:50:29 info: All unit tests passed.
2021-06-15 11:50:31 info: All steps were completed successfully
2021-06-15 11:51:36 info: --------------------------------------------
2021-06-15 11:51:36 info: >>> Running Unit Test `ex4-pokerDiceAll` <<<
2021-06-15 11:51:36 info: --------------------------------------------
2021-06-15 11:51:39 info: All unit tests passed.
2021-06-15 11:51:41 info: All steps were completed successfully
2021-06-15 12:00:30 info: ----------------------------------------------
2021-06-15 12:00:30 info: >>> Running Unit Test `ex5-pokerDiceChain` <<<
2021-06-15 12:00:30 info: ----------------------------------------------
2021-06-15 12:00:30 info: Exercise has not yet been modified
2021-06-15 12:00:34 info: All unit tests passed.
2021-06-15 12:00:36 info: All steps were completed successfully
2021-06-15 12:03:10 info: ----------------------------------------------
2021-06-15 12:03:10 info: >>> Running Unit Test `ex5-pokerDiceChain` <<<
2021-06-15 12:03:10 info: ----------------------------------------------
2021-06-15 12:03:13 info: All unit tests passed.
2021-06-15 12:03:15 info: All steps were completed successfully
2021-06-15 12:38:20 info: ---------------------------------------
2021-06-15 12:38:20 info: >>> Running Unit Test `ex1-johnWho` <<<
2021-06-15 12:38:20 info: ---------------------------------------
2021-06-15 12:38:25 info: All unit tests passed.
2021-06-15 12:38:27 info: All steps were completed successfully
2021-06-15 12:38:35 info: -------------------------------------------------
2021-06-15 12:38:35 info: >>> Running Unit Test `ex2-checkDoubleDigits` <<<
2021-06-15 12:38:35 info: -------------------------------------------------
2021-06-15 12:38:37 info: All unit tests passed.
2021-06-15 12:38:39 info: All steps were completed successfully
2021-06-19 04:31:55 info: ---------------------------------------------
2021-06-19 04:31:55 info: >>> Running Unit Test `ex1-programmerFun` <<<
2021-06-19 04:31:55 info: ---------------------------------------------
2021-06-19 04:32:02 info: All unit tests passed.
2021-06-19 04:32:03 error: *** ESLint Report ***

ex1-programmerFun/index.js
  23:9  warning  Identifier 'json_data' is not in camelCase  hyf/camelcase

âœ– 1 problem (0 errors, 1 warning)


2021-06-19 04:32:34 info: ---------------------------------------------
2021-06-19 04:32:34 info: >>> Running Unit Test `ex1-programmerFun` <<<
2021-06-19 04:32:34 info: ---------------------------------------------
2021-06-19 04:32:39 info: All unit tests passed.
2021-06-19 04:32:41 info: All steps were completed successfully
2021-06-19 10:49:39 info: ------------------------------------------
2021-06-19 10:49:39 info: >>> Running Unit Test `ex2-pokemonApp` <<<
2021-06-19 10:49:39 info: ------------------------------------------
2021-06-19 10:49:44 info: All unit tests passed.
2021-06-19 10:49:45 error: *** ESLint Report ***

ex2-pokemonApp/index.js
  30:3   warning  Remove commented-out code                                                                                   hyf/no-commented-out-code
  33:3   warning  Remove commented-out code                                                                                   hyf/no-commented-out-code
  34:3   warning  Please do not use `innerHTML` in the homework. Use `textContent` and/or `document.createElement()` instead  no-restricted-syntax
  37:10  error    'fetchImage' is defined but never used                                                                      no-unused-vars

âœ– 4 problems (1 error, 3 warnings)


2021-06-21 04:28:32 info: ------------------------------------------
2021-06-21 04:28:32 info: >>> Running Unit Test `ex2-pokemonApp` <<<
2021-06-21 04:28:32 info: ------------------------------------------
2021-06-21 04:28:38 info: All unit tests passed.
2021-06-21 04:28:39 error: *** ESLint Report ***

ex2-pokemonApp/index.js
  33:3   warning  The homework tests do not support `innerText`. Please replace with `textContent`  no-restricted-syntax
  49:40  warning  Results from `map` are unused. Replace `map` with `forEach`                       no-restricted-syntax
  49:40  error    Results from `map` are unused. Replace `map` with `forEach`                       hyf/use-map-result

âœ– 3 problems (1 error, 2 warnings)


2021-06-21 04:29:57 info: ------------------------------------------
2021-06-21 04:29:57 info: >>> Running Unit Test `ex2-pokemonApp` <<<
2021-06-21 04:29:57 info: ------------------------------------------
2021-06-21 04:30:03 info: All unit tests passed.
2021-06-21 04:30:04 error: *** ESLint Report ***

ex2-pokemonApp/index.js
  49:40  warning  Results from `map` are unused. Replace `map` with `forEach`  no-restricted-syntax
  49:40  error    Results from `map` are unused. Replace `map` with `forEach`  hyf/use-map-result

âœ– 2 problems (1 error, 1 warning)


2021-06-21 05:02:47 info: -----------------------------------------
2021-06-21 05:02:47 info: >>> Running Unit Test `ex3-rollAnAce` <<<
2021-06-21 05:02:47 info: -----------------------------------------
2021-06-21 05:02:51 info: All unit tests passed.
2021-06-21 05:02:52 error: *** ESLint Report ***

ex3-rollAnAce.js
  28:30  error  'error' is not defined  no-undef

âœ– 1 problem (1 error, 0 warnings)


2021-06-21 05:03:14 info: -----------------------------------------
2021-06-21 05:03:14 info: >>> Running Unit Test `ex3-rollAnAce` <<<
2021-06-21 05:03:14 info: -----------------------------------------
2021-06-21 05:03:17 info: All unit tests passed.
2021-06-21 05:03:19 info: All steps were completed successfully
2021-06-21 05:11:11 info: ----------------------------------------
2021-06-21 05:11:11 info: >>> Running Unit Test `ex4-diceRace` <<<
2021-06-21 05:11:11 info: ----------------------------------------
2021-06-21 05:11:15 error: *** Unit Test Error Report ***

- ex4-diceRace should resolve as soon as a dice settles successfully
Cannot access 'allDices' before initialization,
- ex4-diceRace should reject with an Error as soon as a dice rolls off the table

2021-06-21 05:11:16 error: *** ESLint Report ***

ex4-diceRace.js
  17:16  error  'dice' is defined but never used  no-unused-vars

âœ– 1 problem (1 error, 0 warnings)


2021-06-21 05:11:32 info: ----------------------------------------
2021-06-21 05:11:32 info: >>> Running Unit Test `ex4-diceRace` <<<
2021-06-21 05:11:32 info: ----------------------------------------
2021-06-21 05:11:36 error: *** Unit Test Error Report ***

- ex4-diceRace should resolve as soon as a dice settles successfully,

2021-06-21 05:11:37 error: *** ESLint Report ***

ex4-diceRace.js
  16:9  error  'allDices' is assigned a value but never used  no-unused-vars

âœ– 1 problem (1 error, 0 warnings)


2021-06-21 05:12:00 info: ----------------------------------------
2021-06-21 05:12:00 info: >>> Running Unit Test `ex4-diceRace` <<<
2021-06-21 05:12:00 info: ----------------------------------------
2021-06-21 05:12:04 info: All unit tests passed.
2021-06-21 05:12:06 info: All steps were completed successfully
2021-06-21 05:15:01 info: ----------------------------------------
2021-06-21 05:15:01 info: >>> Running Unit Test `ex5-vscDebug` <<<
2021-06-21 05:15:01 info: ----------------------------------------
2021-06-21 05:15:01 info: Exercise has not yet been modified
2021-06-21 05:15:01 warn: A unit test file was not provided.
2021-06-21 05:15:03 info: All steps were completed successfully
2021-06-21 06:17:55 info: ----------------------------------------
2021-06-21 06:17:55 info: >>> Running Unit Test `ex4-diceRace` <<<
2021-06-21 06:17:55 info: ----------------------------------------
2021-06-21 06:17:58 info: All unit tests passed.
2021-06-21 06:18:00 info: All steps were completed successfully
2021-06-21 06:21:16 info: ----------------------------------------
2021-06-21 06:21:16 info: >>> Running Unit Test `ex5-vscDebug` <<<
2021-06-21 06:21:16 info: ----------------------------------------
2021-06-21 06:21:16 warn: A unit test file was not provided.
2021-06-21 06:21:18 info: All steps were completed successfully
2021-06-21 07:04:42 info: ----------------------------------------
2021-06-21 07:04:42 info: >>> Running Unit Test `ex5-vscDebug` <<<
2021-06-21 07:04:42 info: ----------------------------------------
2021-06-21 07:04:42 warn: A unit test file was not provided.
2021-06-21 07:04:46 info: All steps were completed successfully
2021-06-21 08:51:15 info: --------------------------------------------
2021-06-21 08:51:15 info: >>> Running Unit Test `ex6-browserDebug` <<<
2021-06-21 08:51:15 info: --------------------------------------------
2021-06-21 08:51:15 warn: A unit test file was not provided.
2021-06-21 08:51:18 info: All steps were completed successfully
2021-06-22 12:38:14 info: ---------------------------------------------
2021-06-22 12:38:14 info: >>> Running Unit Test `ex1-programmerFun` <<<
2021-06-22 12:38:14 info: ---------------------------------------------
2021-06-22 12:38:23 info: All unit tests passed.
2021-06-22 12:38:25 info: All steps were completed successfully
2021-06-22 12:39:38 info: ------------------------------------------
2021-06-22 12:39:38 info: >>> Running Unit Test `ex2-pokemonApp` <<<
2021-06-22 12:39:38 info: ------------------------------------------
2021-06-22 12:39:47 info: All unit tests passed.
2021-06-22 12:39:48 error: *** ESLint Report ***

ex2-pokemonApp/index.js
  49:40  warning  Results from `map` are unused. Replace `map` with `forEach`  no-restricted-syntax
  49:40  error    Results from `map` are unused. Replace `map` with `forEach`  hyf/use-map-result

âœ– 2 problems (1 error, 1 warning)


2021-06-22 12:42:25 info: -----------------------------------------
2021-06-22 12:42:25 info: >>> Running Unit Test `ex3-rollAnAce` <<<
2021-06-22 12:42:25 info: -----------------------------------------
2021-06-22 12:42:30 info: All unit tests passed.
2021-06-22 12:42:33 info: All steps were completed successfully
2021-06-22 12:43:05 info: ----------------------------------------
2021-06-22 12:43:05 info: >>> Running Unit Test `ex4-diceRace` <<<
2021-06-22 12:43:05 info: ----------------------------------------
2021-06-22 12:43:11 info: All unit tests passed.
2021-06-22 12:43:14 info: All steps were completed successfully
